doc/source/using_config.rst

```diff
@@ -178,6 +189,19 @@ Attributes
   * ``auto``: Only cache the build trees where necessary (e.g. for failed builds)
   * ``always``: Always cache the build tree.
 
+* ``storage-service``
```

Comment:
I'm a bit unsatisfied with the documentation... without looking deeply into the implementation details, it is unclear to me by reading this:

* What kind of service is sufficient to specify here ?
  * More concretely, when we describe the `storage-service` attribute of the remote execution configuration, we specify that that CAS service is compatible with the *storage* service offered by *cache servers* (linking back to the cache server area)...
  * I think this should clarify if this is indeed the same service, and whether a service with a *storage* implementation is sufficient, or whether the service must also implement the *index* functionality
  * If indexing is required for this service, then we really should have a way to use this feature without hard requiring the usage of a single service which implements both *storage* and *index* functionalities.
* If one uses remote execution at all, then is it recommended that this configuration be configured to use the *same* `storage-service` as the one which you configure as a part of remote execution ?

Reply:
For the second point above, I see that remote execution will fallback on this service, and it can be left unspecified in a remote execution configuration, that sort of answers my question.

In the following text here where we say that this is especially (but interestingly not *uniquely*) useful in combination with remote execution, we might do well to also mention this detail, possibly with some kind of addition to the phrase like *"... and will be used as a default for remote execution `storage-service`"*. Or, perhaps a `.. tip::` block would be better to illustrate this (and might avoid overly wordy sentences).

Reply:
It is compatible with the *storage* service offered by cache servers, no index functionality is needed. I've added a sentence to clarify this.

I've also added a `.. tip::` block as suggested.

Does the documentation look ok now?